cmd/eval_test.go

```diff
@@ -1866,8 +1867,8 @@ time.clock(input.y, time.clock(input.x))
 			if err != nil {
 				t.Fatal("unexpected error:", err)
 			}
-			if actual := buf.String(); actual != tc.expected {
-				t.Errorf("expected output %q\ngot %q", tc.expected, actual)
+			if diff := cmp.Diff(buf.String(), tc.expected); diff != "" {
+				t.Log("output mismatch (-want +got):\n", diff)
```

Comment:
`cmp.Diff()` is a nice improvement to readability 👍, but we still want the test to fail on mismatch.

```suggestion
				t.Error("output mismatch (-want +got):\n", diff)
```

Same in a couple of other places.